core/libs/commonwealth/commonwealth/mavlink_comm/MavlinkComm.py

```diff
@@ -17,7 +17,7 @@ def __init__(self) -> None:
         self.system_id = int(os.environ.get("MAV_SYSTEM_ID", 1))
         self.component_id = int(os.environ.get("MAV_COMPONENT_ID_ONBOARD_COMPUTER4", 194))
         self.sequence = 0
-        self.m2r_address = "localhost:8088"
+        self.m2r_address = "localhost:6040"
```

Comment:
wait.. what is going on here? was this working before?
Can you explain the reasoning in the commit body?

Reply:
This had 8088 as default before (I think it was the old M2R default port), while the VehicleManager instance would change it to 6040 (the actual port we use).
I changed the default so there's no need to set it on each instantiation.
Will put this comment on the commit.